test/verify/check-login

```diff
@@ -267,6 +271,7 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         self.allow_journal_messages('Error executing command as another user: Not authorized',
                                     'This incident has been reported.',
                                     'sudo: a password is required',
+                                    'sudo: Account or password is expired, reset your password and try again',
```

Comment:
Why is this new message here?

Reply:
Ah, I should have said that -- this has always been there, just a race/flake as the test VM shuts down fast enough. I just noticed it when running the test many times in a row. This could/should have been its own commit indeed.

Reply:
Move to its own commit now.